content/Consumer Privacy Regulations CCPA and GDPR - Pablo Blanco.md

```diff
@@ -0,0 +1,240 @@
+policies and processes necessary for compliance and to analyze the current and
+future impact it will have in comparison to GDPR.
+
+This California state law establishes new consumer rights relating to the access
+of personal information that is held by businesses. The meaning of Personal
+information is quite broad, the definition given in
+[CCPA](https://oag.ca.gov/privacy/) says:
+
+_Personal information is information that identifies, relates to, or could
+reasonably be linked with an individual or household._
+
+Some examples of that are customer name, social security number, email address,
+phone number, records of products purchased, internet browsing history
+(including cookies), geolocation data, fingerprints, IP and MAC address (or any
+device identifier), among others. Personal information does not include publicly
+available information from federal, state, or local government records.
+
+Businesses have a track record of using personal information to benefit their
+agenda, CCPA will serve to protect California consumer rights and encourage
+stronger privacy, traceability, and greater transparency overall. CCPA allows
+consumers to have control, ownership, gain security over their personal
+information, and also the ability to request that any business disclose (and
+delete) the personal information collected.
+
+This data protection gives consumers the right to:
+
+- Know what personal information is being collected
+- Access the personal information that is collected, and request its deletion
+- Know if their personal information is being shared, and if so, with whom
+  opt-out of the sale of their personal information
+- Have equal service and price, whether or not they choose to exercise their
+  privacy rights (right to non-discrimination).
+
+It’s also prohibited selling the personal information of consumers ages 13–16
+(unless the consumer opts-in). For consumers under the age of 13, consent from a
+parent is a must.
+
+## Who does the CCPA apply to?
+
+CCPA defines a business as a for-profit entity that collects consumer personal
+data and applies to all companies that serve California residents, whether they
+are physically placed in California or not. If the company meets at least one of
+the following thresholds, may be subject to compliance:
+
+- Companies with at least \$25 million in annual revenue (applies to total
+  revenue, not just revenue in California).
+- Companies that annually buy, sell, receive, or share personal information of
+  at least 50,000 consumers, homes, or devices for commercial purposes.
+- Companies that derive 50% or more of their annual revenue from the sale of
+  personal information.
+
+## Some differences between CCPA and GDPR
+
+The European General Data Protection Regulation is an evolution of the EU’s
+existing data rules and replaced the Data Protection Directive (DPD). It
+addresses many of the lacks in the DPD, including adding requirements for
```

Comment:
`lacks` sound a little weird in plural, what about `deficiencies`, `weaknesses`?